e2e/docker/README.md

```diff
@@ -0,0 +1,15 @@
+# Flower containers testing
+
+This directory is used to test Flower containers in a minimum scenario, that is, with 2 clients and without HTTPS. The FL setup uses PyTorch, the CIFAR10 dataset, and a CNN. This is mainly to test the functionalities of Flower in a containerized architecture. It can be easily extended to test more complex communication set-ups.
+
+It uses a subset of size 1000 for the training data and 10 data points for the testing.
+
+To execute locally, run the following in CLI:
+``` shell
+$ docker compose up -d --remove-orphans --force-recreate --build
```

Comment:
If we want to make sure we are always using the latest `nightly` version, we need to run the following:

```
# pulls the latest supernode and serverapp nightly
docker compose build --pull 
# pulls the latest superlink nightly
docker compose up -d --remove-orphans --force-recreate --pull always 
```

Reply:
Nice! Will update it.